k8s/prod/common/sscs/sscs-evidence-share.yaml

```diff
@@ -33,12 +26,7 @@ spec:
         PDF_SERVICE_BASE_URL: https://docmosis.platform.hmcts.net/rs/render
         PDF_SERVICE_HEALTH_URL: https://docmosis.platform.hmcts.net/rs/status
         IDAM_OAUTH2_REDIRECT_URL: https://sscs-case-loader-prod.service.core-compute-prod.internal
-        JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
```

Comment:
this still needs to be here until you update your base docker image version:
https://github.com/hmcts/sscs-evidence-share/blob/master/Dockerfile#L3